scanners/sslyze/README.md

```diff
@@ -2,6 +2,9 @@
 title: "SSLyze"
 path: "scanners/sslyze"
 category: "scanner"
+type: "Kubernetes"
```

Comment:
I'm not quite sure why the *type* needs to be _Kubernetes_ and cannot be more general

Reply:
I think this is a typo / copy paste error.
Type should have probably been "SSL". (probably better to call it tls...)
And state should be "released"

Reply:
yep, that a typo